packages/lesswrong/components/Layout.tsx

```diff
@@ -238,51 +238,53 @@ class Layout extends PureComponent<LayoutProps,LayoutState> {
       }}>
       <TableOfContentsContext.Provider value={this.setToC}>
         <div className={classNames("wrapper", {'alignment-forum': getSetting('forumType') === 'AlignmentForum'}) } id="wrapper">
-          <DialogManager>
-            <CommentBoxManager>
-              <CssBaseline />
-              <Helmet>
-                <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
-                <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.0.0/themes/reset-min.css"/>
-                <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500"/>
-                { theme.typography.fontDownloads &&
-                    theme.typography.fontDownloads.map(
-                      (url)=><link rel="stylesheet" key={`font-${url}`} href={url}/>
-                    )
-                }
-                <meta httpEquiv="Accept-CH" content="DPR, Viewport-Width, Width"/>
-                <link rel="stylesheet" href="https://use.typekit.net/jvr1gjm.css"/>
-              </Helmet>
-              
-              <Components.AnalyticsClient/>
-              <Components.NavigationEventSender/>
+          <Components.AnalyticsPageInitializer>
```

Comment:
Does this need to be a wrapper, as opposed to a no-children component deeper in the tree?

Reply:
Good point, it doesn't need to be. I'll change that.

Reply:
Changed.